src/platform_impl/windows/event_loop.rs

```diff
@@ -1510,8 +1519,17 @@ unsafe extern "system" fn public_window_callback<T>(
                 // The information retrieved appears in reverse chronological order, with the most recent entry in the first
                 // row of the returned array
                 for pointer_info in pointer_infos.iter().rev() {
-                    let x = pointer_info.ptPixelLocation.x as f64;
-                    let y = pointer_info.ptPixelLocation.y as f64;
+                    let mut location = POINT {
+                        x: pointer_info.ptPixelLocation.x,
+                        y: pointer_info.ptPixelLocation.y,
+                    };
```

Comment:
This may be better-suited to another PR, but would it be possible to use `ptHimetricLocation` to get sub-pixel precision when handling this event? I should've caught that in the original PR, but I didn't look closely enough and didn't realize it was quite possible.

Reply:
Sure, i will look at it. I can open a new PR if you wish.

Reply:
Do whatever works best for you. Either option is fine by me.

Reply:
I've implemented it in a separate branch so i'll open a separate PR for it.